src/app/preference/preference-modal/preference-modal.component.html

```diff
@@ -0,0 +1,77 @@
+<ion-content color="light" class="pop-up-content test" text-center>
+  <ng-container *ngIf="!showUpdateModal; else showUpdateModalTemplate">
+    <ion-toolbar class="pn-title-toolbar">
+      <ion-icon name="close"
+        (click)="close()"
+        color="primary"
+        size="large"
+        slot="end"
+        margin-start
+        class="close-modal"
+      ></ion-icon>
+      <ion-title class="ion-text-center pm-title">Notifications</ion-title>
+    </ion-toolbar>
+  
+    <ng-container *ngIf="(preferences$ | async ) as preferences; else noPreferenceAvailable">
+      <ion-list *ngFor="let cat of preferences.categories">
+        <ion-list-header class="no-bg preference-cat-item">{{cat.name}}</ion-list-header>
+  
+        <ion-item *ngFor="let pref of cat.preferences" (click)="PreferenceUpdateModal($event,pref.key)">
+          <ion-label>
+            <h3 class="preference-header-item">{{pref.name}}</h3>
+            <app-options [options]="pref.options"></app-options>
+          </ion-label>
+          <ion-icon name="chevron-forward-outline" color="medium" slot="end"></ion-icon>
+        </ion-item>
+      </ion-list>
+    </ng-container>
+  
+    <ng-template #noPreferenceAvailable>
+      <div class="ion-text-center ion-margin-bottom">
+        <p>No preferences setting available.</p>
+      </div>
+    </ng-template>
+  </ng-container>
+ 
+  
+  <ng-template #showUpdateModalTemplate>
+    <ion-header mode="ios">
```

Comment:
it's should be the same for this  `#showUpdateModalTemplate` part.
this part using the entire same codebase as in [preference-update.component.html](https://github.com/intersective/app-v2/blob/AV2-846/Notification-preference-desktop/src/app/preference/preference-update/preference-update.component.html)

try practise [template syntax](https://angular.io/guide/architecture-components#template-syntax)

p/s: anything that's repeated, should be made as shareable "component". Next time, we dont need to edit multiple time/components for one similar change request.